src/test/java/com/ibm/eventstreams/connect/jdbcsink/database/writer/JDBCWriterTest.java

```diff
@@ -0,0 +1,129 @@
+package com.ibm.eventstreams.connect.jdbcsink.database.writer;
+
+import org.apache.kafka.connect.data.*;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.Test;
+
+import com.ibm.db2.jcc.am.DatabaseMetaData;
+import com.ibm.eventstreams.connect.jdbcsink.database.datasource.IDataSource;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.Collections;
+
+import static org.mockito.Mockito.*;
+
+public class JDBCWriterTest {
+
+        @Test
+        public void testCreateTable_TableDoesNotExist_CreatesTable() throws SQLException {
+                // Prepare test data
+                Connection connection = mock(Connection.class);
+                IDataSource dataSource = mock(IDataSource.class);
+                PreparedStatement preparedStatement = mock(PreparedStatement.class);
+                when(dataSource.getConnection()).thenReturn(connection);
+                when(connection.prepareStatement("CREATE TABLE ? (?)")).thenReturn(preparedStatement);
+                // when(connection.getMetaData()).thenReturn(mock(DatabaseMetaData.class));
+
+                String tableName = "test_table";
+
+                Schema schema = SchemaBuilder.struct()
+                                .field("id", Schema.INT32_SCHEMA)
+                                .field("name", Schema.STRING_SCHEMA)
+                                .build();
+
+                JDBCWriter jdbcWriter = new JDBCWriter(dataSource);
+
+                // Execute the method under test
+                jdbcWriter.createTable(connection, tableName, schema);
+
+                // Verify the behavior
+                verify(connection).prepareStatement("CREATE TABLE ? (?)");
+                verify(preparedStatement).execute();
+                verify(preparedStatement).close();
```

Comment:
How do these verify statements act as assertions? I may have mis understood something here

Reply:
org.mockito.Mockito.verify Verifies certain behavior happened once.

Alias to verify(mock, times(1)) E.g:

verify(mock).someMethod("some arg");
Above is equivalent to:

verify(mock, times(1)).someMethod("some arg");

Reply:
org.mockito.Mockito.verify Verifies certain behavior happened once.

Alias to verify(mock, times(1)) E.g:

verify(mock).someMethod("some arg");
Above is equivalent to:

verify(mock, times(1)).someMethod("some arg");